pyvera/__init__.py

```diff
@@ -293,13 +307,7 @@ def set_value(self, name, value):
                         'action': 'Set' + name,
                         'newLoadlevelTarget': value}
                 else:
-                    payload = {
```

Comment:
Some docs here http://wiki.micasaverde.com/index.php/Luup_UPnP_Variables_and_Actions although just claims to be the dimmer!

Still reading...